codex-processes-ap2-docker-test-setup/data/feasibility-bundle.json

```diff
@@ -45,6 +45,17 @@
             },
             "valueString": "requestSimpleFeasibilityMessage"
           },
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://highmed.org/fhir/CodeSystem/bpmn-message",
+                  "code": "business-key"
+                }
+              ]
+            },
+            "valueString": "urn:uuid:600db7d6-ff7c-4545-b21e-2f6a371bddc1"
```

Comment:
This doesn't need to be a URI. So I would only use the plain UUID here.

Reply:
Has been adjusted, thanks.